X/Xyce/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Xyce"
+version = v"7.2.0"
+
+# Collection of sources required to complete build
+sources = [
+            GitSource("https://github.com/Xyce/Xyce.git", "a61faef4bfb2f36f1aa7cc44264bbbb66fbaac11"),
+            DirectorySource("./bundled")
+          ]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+apk add flex-dev
+cd Xyce
+if [[ $target != i686-linux-gnu ]] && [[ $target != x86_64-linux-gnu ]] ; then
+        atomic_patch -p1 ${WORKSPACE}/srcdir/patches/cross.patch
+fi
+./bootstrap
+cd ..
+mkdir buildx
+cd buildx
+/workspace/srcdir/Xyce/./configure --enable-shared --disable-mpi --prefix=${prefix} LDFLAGS="-L${libdir} -lopenblas" CPPFLAGS="-I/$prefix/include -I/usr/include" --host=${target}
```

Comment:
```suggestion
/workspace/srcdir/Xyce/./configure --enable-shared --disable-mpi --prefix=${prefix} LDFLAGS="-L${libdir} -lopenblas" CPPFLAGS="-I/${includedir} -I/usr/include" --host=${target}
```
why `/usr/include`?  That's a host directory

Reply:
`apk add flex-dev` adds `FlexLexer.h` at `/usr/include`. Wizard throws an error if it isn't explicitly specified